automatic/1password/update.ps1

```diff
@@ -53,18 +53,22 @@ function Get-LatestOPW {
   }
 }
 
-$releases_opw4 = 'https://app-updates.agilebits.com/download/OPW4'
-$kind_opw4 = '1password4'
-$releases_opw = 'https://app-updates.agilebits.com/download/OPW7/Y'
-$kind_opw = '1password'
+$releases = 'https://downloads.1password.com/win/1PasswordSetup-latest.exe'
```

Comment:
Would change to `$releases = https://downloads.1password.com/win/1PasswordSetup-8.8.0.exe`